lib/logstash/outputs/exec.rb

```diff
@@ -50,12 +50,12 @@ def receive(event)
 
     Open3.popen3(cmd) do |stdin, stdout, stderr|
       if @logger.debug?
-        @logger.pipe(stdout => :debug, stderr => :debug)
+        @logger.debug(stdout => :debug, stderr => :debug)
```

Comment:
should it be something like this? (I did not test it)

```ruby
 @logger.debug(:stdout => stdout.read, :stderr => stderr.read)
 ```
The stdout and the stderr should be IO object and we need to read them to display the messages.